docs/Joining-Data-(v0.25.0+).md

```diff
@@ -0,0 +1,117 @@
+by [opening an issue](https://github.com/brimdata/brim/wiki/Troubleshooting#opening-an-issue)
+or reaching out on the [Brim public Slack](https://www.brimsecurity.com/join-slack/).
+We'd also love to hear your success stories and variations, so please don't be
+shy!
+
+# Example Usage
+
+By its nature, a join operation requires multiple inputs that will
+ultimately be combined. The Zed [`join` docs](https://github.com/brimdata/zed/tree/main/docs/language/operators#join)
+show examples with the [Zed CLI tools](https://github.com/brimdata/zed/blob/main/cmd/zed/README.md)
+that specify these inputs as named files or Pools in a [Zed Lake](https://github.com/brimdata/zed/blob/main/docs/lake/design.md).
+
+Brim release `v0.25.0` includes initial support for storing data in Zed Lakes.
+However, due to a current limitation ([brim/1618](https://github.com/brimdata/brim/issues/1618)),
+Zed queries issued from within Brim cannot yet access multiple Pools
+simultaneously. Instead such queries are currently limited to accessing the
+data from whichever Pool is currently selected from the **Pools** list.
+
+Because of this limitation, the [streamed input example](https://github.com/brimdata/zed/tree/main/docs/language/operators#example-5---streamed-input)
+is the only one shown that can currently be executed as is from within Brim.
+If the example `fruit.ndjson` and `people.ndjson` was already present in a
+single Pool (such as if concatenated into a single file and dragged into the
+app), we can see the same query output as shown in the doc.
+
+![Streamed Join Example](media/Join-Streamed.png)
+
+> **Note:** Because the data being displayed is of multiple different shapes,
+> Brim's **Columns** setting has been modified here to set **Headers: On**.
+
+While queries executed from inside Brim are subject to this limitation, the
+Zed backend that's launched behind Brim is not. Therefore if you're willing to
+perform joins at the shell using [`zapi`](https://github.com/brimdata/zed/blob/main/cmd/zed/README.md#zapi)
+you can execute all the other examples shown while accessing data from multiple
+Pools. The joined results can be sent into yet another Pool for further query
+from within Brim, if desired.
+
+To illustrate this, we'll walk through the [example that shows inputs from Pools](https://github.com/brimdata/zed/tree/main/docs/language/operators#example-4---inputs-from-pools).
+To ensure API-compatibility with the Zed backend, we'll use the `zapi` binary
+found in the `zdeps` directory under the Brim [application binaries](https://github.com/brimdata/brim/wiki/Filesystem-Paths#application-binaries-v0250)
+path, specifically on macOS in this case.
+
+Because Brim already takes care of initializing the Zed Lake, we pick up the
+example commands with the creation and population of the separate Pools
+
+```
+$ export ZDEPS="/Applications/Brim.app/Contents/Resources/app.asar.unpacked/zdeps"
```

Comment:
Maybe do `PATH=/Applications/Brim.app/Contents/Resources/app.asar.unpacked/zdeps:$PATH` instead so you don't need the `$ZDEPS/` prefix?

Reply:
@nwt: Yeah, sounds good. I've pushed a commit to set `PATH` instead of the `$ZDEPS/` prefix.